java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -705,6 +705,7 @@ private void buildSwitchStatement(SwitchStatementTree switchStatementTree) {
         if (!hasDefaultCase) {
           hasDefaultCase = containsDefaultCase(caseGroupTree.labels());
         }
+        currentBlock.elements.add(caseGroupTree);
```

Comment:
@m-g-sonar Are you ok to add `caseGroupTree` as a block element in the CFG? (See also the tests below) Is it the right order of the elements in the block?

Reply:
Worth having a comment to mention why we need it. If it's simplify implementation of the rule, let's document it.